packages/manager/modules/pci/src/projects/project/ai/apps/add/add.controller.js

```diff
@@ -373,4 +373,9 @@ export default class AppAddController {
         this.isAdding = false;
       });
   }
+
+  goToUserTokens() {
+    this.trackApps(`banner-manage-users-tokens`);
```

Comment:
This value is used more than once so you can put it in constants file to avoid duplicating it